src/main/java/com/openlattice/datastore/services/EdmService.java

```diff
@@ -311,6 +311,24 @@ public void createEntityType( EntityType entityTypeRaw ) {
         return propertyTypes.keySet();
     }
 
+    private void setupDefaultEntityTypePropertyMetadata( UUID entityTypeId ) {
+        final var et = getEntityType( entityTypeId );
+        final var propertyTags = et.getPropertyTags();
+
+        Map<UUID, PropertyType> propertyTypeMap = getPropertyTypesAsMap( et.getProperties() );
+        et.getProperties().forEach( propertyTypeId -> {
+            EntityTypePropertyKey key = new EntityTypePropertyKey( entityTypeId, propertyTypeId );
+            PropertyType property = propertyTypeMap.get( propertyTypeId );
+            EntityTypePropertyMetadata metadata = new EntityTypePropertyMetadata(
+                    property.getTitle(),
+                    property.getDescription(),
+                    new LinkedHashSet(propertyTags.get(propertyTypeId)),
+                    true
+                    );
+            entityTypePropertyMetadata.put( key, metadata );
```

Comment:
This isn't going to be that expensive since we're not creating entity types all the time / they typically don't have a huge number of property types, but this call can also be bulked by instantiating a `Map<EntityTypePropertyKey, EntityTypePropertyMetadata>` of size (# property types in entity set), adding the values to that map, and then after exiting the `.forEach`, calling `entityTypePropertyMetadata.putAll( <yourMap> )` -- not a huge deal but would speed up entity type creation a little bit, and is a little bit safer in case something goes wrong / throws an exception halfway through the forEach (so that you don't end up in a weird halfway state)